python/pyarrow/tests/test_convert_pandas.py

```diff
@@ -1154,20 +1152,21 @@ def test_fixed_offset_timezone(self):
         _check_pandas_roundtrip(df)
         _check_serialize_components_roundtrip(df)
 
+# ----------------------------------------------------------------------
+# Conversion tests for string and binary types.
 
-class TestConvertStringLikeTypes(object):
-    """
-    Conversion tests for string and binary types.
-    """
 
-    def test_unicode(self):
-        repeats = 1000
-        values = [u'foo', None, u'bar', u'mañana', np.nan]
-        df = pd.DataFrame({'strings': values * repeats})
-        field = pa.field('strings', pa.string())
-        schema = pa.schema([field])
+def test_pandas_unicode():
```

Comment:
Why not leave this in `TestConvertStringLikeTypes`?

Reply:
Moved it back. We should address de-unittest-ing the tests in a different PR if we want to do that